src/design-tokens/ThemeProvider.tsx

```diff
@@ -9,7 +9,8 @@ import ThemeContext from './ThemeContext';
 import useLocalStorage from './useLocalStorage';
 
 const ThemeProvider: React.FC = ({ children }) => {
-  const isDarkModeDefault = window?.__VERDACCIO_BASENAME_UI_OPTIONS?.darkMode;
+  const prefersDarkMode = window?.matchMedia('(prefers-color-scheme:dark)').matches;
```

Comment:
```suggestion
   const prefersDarkMode = window.matchMedia?.('(prefers-color-scheme:dark)').matches;
```

after this fix, the tests will pass 😉... Good Job!! 👏